web/libs/editor/src/mixins/AreaMixin.js

```diff
@@ -155,9 +158,17 @@ export const AreaMixinBase = types
     },
   }))
   .volatile(() => ({
-    // selected: false,
+    // index of the region in the regions tree (Outliner); will be updated on any order change
+    region_index: null,
   }))
   .actions((self) => ({
+    setCurrentIndex(index) {
```

Comment:
Could it be better to name it `setRegionIndex`? We have `setItemIndex` in `RegionsMixin` to indicate the object it relates to. So it can be confusing.

Reply:
the idea was to indicate that this region is dynamic. On the other hand if we have a setter then it's probably dynamic by nature, huh. I'll change it, thank you